spec/models/course_spec.rb

```diff
@@ -94,6 +94,66 @@
         end
       end
 
+      describe '#current_level_threshold' do
+        subject do
+          -> (course, experience_points) { course.current_level_threshold experience_points }
+        end
+
+        context 'when there are no levels created in the course' do
+          let!(:course_without_levels) { create(:course) }
+          it 'returns the correct threshold value' do
+            expect(subject[course_without_levels, 0]).to eq 0
+            expect(subject[course_without_levels, 1]).to eq 0
```

Comment:
I'll change it to `.to eq(0)` for now, left out fixing this part of rspec.

Both are fine to me - if anything, no parentheses makes it more readable, but contributors will need some time to figure out our codebase's conventions.